src/main/java/com/hibernate/Main.java

```diff
@@ -1,21 +1,35 @@
 package com.hibernate;
 
 import com.hibernate.lib.Injector;
+import com.hibernate.model.Movie;
+import com.hibernate.model.MovieSession;
 import com.hibernate.model.User;
 import com.hibernate.service.AuthenticationService;
-import com.hibernate.util.HashUtil;
+import com.hibernate.service.MovieService;
+import com.hibernate.service.MovieSessionService;
+import com.hibernate.service.ShoppingCartService;
 
 public class Main {
     private static Injector injector = Injector.getInstance("com.hibernate");
     private static final AuthenticationService authService =
             (AuthenticationService) injector.getInstance(AuthenticationService.class);
+    private static final ShoppingCartService SHOPPING_CART_SERVICE =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+    private static final MovieSessionService movieSessionService =
+            (MovieSessionService) injector.getInstance(MovieSessionService.class);
+    private static final MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
+    private static final User user = authService.register("123", "123");
```

Comment:
let's make this a local variable

Reply:
Checkstyle dropped error when user was local variable

Reply:
![image](https://user-images.githubusercontent.com/75170409/106872782-5af86a80-66dc-11eb-8bde-4f22fe84fa98.png)

Reply:
let's create it closer to its first usage then?